data.sql

```diff
@@ -1,5 +1,7 @@
-/* Populate database with sample data. */
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Argon','2020-02-03',0,TRUE,'10.23');
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg)
+VALUES('Gabumon','2018-11-15',2,TRUE,8.0),
+      ('Pickachu','2021-01-7',1,FALSE,15.04),
+      ('Devimon','2017-05-12',5,TRUE,11.00);
```

Comment:
- [ ] As [the project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#project-requirements) suggest you should also insert the values of `Agumon`

Reply:
Great insight @AlphaNtihinduka. I will get on this and fix it, thank you.